pkg/controller/master/setting/ntp_server.go

```diff
@@ -42,6 +43,7 @@ func (h *Handler) syncNTPServer(setting *harvesterv1.Setting) error {
 				return err
 			}
 
+			ntpServers := reGenerateNTPServers(ntpSettings, []string{})
```

Comment:
How about just refactoring `util.GetNTPServers`? This also doesn't make `util.GetNTPServers` an unused function. We can also consider moving the call out of the loop so it only run once.

Reply:
Move to utils.
Also, reduce the duplicated re-generate operation when we do not have nodeconfig.